javascript-timers/interval.js

```diff
@@ -0,0 +1,16 @@
+setInterval(bowie, 1000);
```

Comment:
make sure you are saving the `intervalId` into a variable and passing it to `clearInterval` when you are done with the interval